addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1868,6 +1872,40 @@ QUnit.test('chat window does not fetch messages if hidden', async function (asse
     );
 });
 
+QUnit.test('chat window: click on chat header for channel type "chat", redirect to user form', async function (assert) {
+    assert.expect(4);
+
+    this.data['res.partner'].records.push({ id: 7, name: "Demo" });
+    const chat = {
+        channel_type: "chat",
+        is_minimized: true,
+        members: [this.data.currentPartnerId, 7],
+    };
+    this.data['mail.channel'].records.push(chat);
+    await this.start({
+        mockRPC(route, args) {
+            if (args.method === "search_read") {
+                assert.step('partner:search_read');
+                assert.deepEqual(
+                    args.kwargs.domain,
+                    [["user_partner_id", "=", 7]],
+                    'User partner id should be 7'
+                );
+            }
+            return this._super(...arguments);
+        },
+    });
+
+    const chatWindowHeaderName = document.querySelector(`:scope .o_ChatWindow_header .o_ChatWindowHeader_name`);
+    assert.ok(
+        chatWindowHeaderName.classList.contains('o_Clickable_ChatWindowHeaderName'),
+        "thread name should be clickable"
+    );
+    // click on chat window header name
+    chatWindowHeaderName.click();
+    assert.verifySteps(['partner:search_read']);
```

Comment:
This is too technical, maybe we can listen on the `do_action` for the form view instead (see other tests doing it)? Also missing assert comment.